src/OctoshiftCLI.Tests/gei/Commands/GenerateMannequinCsvCommandTests.cs

```diff
@@ -0,0 +1,180 @@
+            mockGithubApi.Setup(x => x.GetMannequins(githubOrgId).Result).Returns(Array.Empty<Mannequin>());
+
+            string csvContent = null;
+
+            var command = new GenerateMannequinCsvCommand(
+                TestHelpers.CreateMock<OctoLogger>().Object,
+                mockTargetGithubApiFactory.Object)
+            {
+                WriteToFile = (_, contents) =>
+                {
+                    csvContent = contents;
+                    return Task.CompletedTask;
+                }
+            };
+
+            var expected = "login,claimantlogin" + Environment.NewLine;
+
+            // Act
+            await command.Invoke("octocat", "unit-test-output", false);
+
+            // Assert
+            csvContent.Should().Be(expected);
+        }
+
+        [Fact]
+        public async Task Mannequins_GenerateCSV_UnreclaimedOnly()
+        {
+            const string githubOrg = "FooOrg";
+            var githubOrgId = Guid.NewGuid().ToString();
+
+            var mannequinsResponse = new[]
+            {
+                new Mannequin
+                {
+                    Id = "monaid",
+                    Login = "mona"
+                },
+                new Mannequin
+                {
+                    Id = "monalisaid",
+                    Login = "monalisa",
+                    MappedUser = new Claimant
+                    {
+                        Id = "monalisamapped-id",
+                        Login = "monalisa_gh"
+                    }
+                }
+            };
+
+
+            var mockGithubApi = TestHelpers.CreateMock<GithubApi>();
+            var mockTargetGithubApiFactory = new Mock<ITargetGithubApiFactory>();
+            mockTargetGithubApiFactory.Setup(m => m.Create(It.IsAny<string>(), It.IsAny<string>())).Returns(mockGithubApi.Object);
+            mockTargetGithubApiFactory.Setup(m => m.Create(It.IsAny<string>(), It.IsAny<string>())).Returns(mockGithubApi.Object);
+
+            mockGithubApi.Setup(x => x.GetOrganizationId(githubOrg).Result).Returns(githubOrgId);
+            mockGithubApi.Setup(x => x.GetMannequins(githubOrgId).Result).Returns(mannequinsResponse);
+
+            string csvContent = null;
+
+            var command = new GenerateMannequinCsvCommand(
+                TestHelpers.CreateMock<OctoLogger>().Object,
+                mockTargetGithubApiFactory.Object)
+            {
+                WriteToFile = (_, contents) =>
+                {
+                    csvContent = contents;
+                    return Task.CompletedTask;
+                }
+            };
+
+            var expected = "login,claimantlogin" + Environment.NewLine
+                + "mona," + Environment.NewLine;
+
+            // Act
+            await command.Invoke(githubOrg, "unit-test-output", false);
+
+            // Assert
+            csvContent.Should().Be(expected);
+            mockGithubApi.Verify(x => x.GetMannequins(githubOrgId));
```

Comment:
we usually don't call verify in situations like this, the only way it could have gotten the expected result is to call getmannequins with the proper orgid, so not really any need to call verify.